homepage/homepage-strings-pt-br.json

```diff
@@ -0,0 +1,71 @@
+  "We asked a few what Gitter means to them, this is what they had to say. If you love Gitter, Tweet us @gitchat and we'll update these quotes every month": "Perguntamos para alguns usuários o que o Gitter significa para eles, e isso é o que eles disseram. Se você também ama o Gitter, tweet para @githcat e nós atualizaremos essas frases todo mês",
+  "Personal pricing": "Preço por pessoa",
+  "Free": "Grátis",
+  "Forever": "Para sempre",
+  "Unlimited": "Ilimitado",
+  "public rooms": "salas públicas",
+  "private room": "salas privadas",
+  "weeks": "semanas",
+  "chat history": "histórico",
+  "Get started for free": "Comece grátis",
+  "Individual": "Individual",
+  "monthly": "mensal",
+  "Unlimited public rooms": "Salas Públicas Ilimitadas",
+  "Unlimited private rooms": "Salas Privadas Ilimitadas",
+  "Unlimited chat history": "Hitórico de conversas Ilimitado",
+  "Purchase now": "Compre agora",
+  "Organisation pricing": "Preço para Empresas",
+  "Bronze": "Bronze",
+  "per user/month": "por usuário/mês",
+  "months": "meses",
+  "Silver": "Prata",
+  "Our free plan includes one private room per organisation you belong to on GitHub": "Nosso plano gratuito inclui uma Sala Privada por Organização que você pertença no Github",
+  "Our Individual plan allows you to create private rooms using your personal account as the owner": "Nosso plano Individual permite que você crie salas privadas usando sua conta pessoal como 'dono da sala'",
+  "Our organisation plans allow you to create private rooms using your GitHub organisation as the owner": "Nossos Planos Empresariais permitem que você crie salas privadas usando sua Organização do GitHub como 'dona da sala'",
+  "Prices may be subject to VAT. By signing up you agree to our": "Preços estão sujeitos a Taxas (impostos), de acordo com a legislação de seu País. Ao cadastrar-se, você concorda com o nosso",
+  "Get Gitter now!": "Obtenha o Gitter agora!",
+  "Gitter is available in all modern browsers as well as apps for desktops and mobile phones": "Gitter está disponível em todos os navegadores modernos, bem como em aplicativo Desktop e para celulares/tablets",
+  "We currently have apps for OSX, iOS, Android and Windows with a Linux app coming soon. We also have a sweet little IRC bridge": "Atualmente, temos aplicativos para OSX, iOS, Android e Windows. Uma versão para Linux estará chegando em breve. Também temos uma pequena 'ponte' para canais IRC",
+  "Mac": "Mac",
+  "Windows": "Windows",
+  "Coming soon": "Em breve",
+  "iPhone": "iPhone",
+  "Android": "Android",
+  "By developers, for developers": "Por desenvolvedores, para desenvolvedores",
+  "Gitter is built in London by a team of developers just like you. Hell even the product guy writes code, although we generally end up rewriting those bits": 
+  "Gitter é feito em Londres por um time de desenvolvedores como você. E cara, mesmo o responsável pelo produto escreve código, embora geralmente nós acabamos re-escrevendo esses bits.",
```

Comment:
Sorry... I was unable on the last days.
@waltfy  Acho que "até" fica bem melhor que "mesmo". Ótimo! :+1: